src/cmd-buildextend-gcp

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# pylint: disable=C0103
+"""
+An operation that mutates a build by uploading to GCP,
+extending the meta.json with GCP image name.
+"""
+# NOTE: PYTHONUNBUFFERED is set in cmdlib.sh for unbuffered output
+
+import argparse
+import json
+import os
+import shutil
+import sys
+
+cosa_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, cosa_dir)
+# pylint: disable=C0413
+from cmdlib import (
+        run_verbose,
+        sha256sum_file,
+        write_json)
+
+# Parse args and dispatch
+parser = argparse.ArgumentParser()
+parser.add_argument("--build", help="Build ID", required=True)
+parser.add_argument("--bucket", help="Storage account to write image to",
+                    default=os.environ.get("GCP_BUCKET", ""), required=True)
```

Comment:
Hmm, we don't need a `default=` if `required=True` is specified, right? That default will just never kick in. Similarly for the other two down there.

Reply:
Indeed, its actually a bit worse...I need to add checking after the fact. Will push a fix shortly.